openbb_terminal/helper_funcs.py

```diff
@@ -1805,3 +1805,28 @@ def load_json(path: str) -> Dict[str, str]:
         )
         console.print(f"[red]{e}[/red]")
         return {}
+
+
+def check_df_empty_or_none(obj) -> bool:
```

Comment:
The less common code the better.

I would put that in the model or just use the instructions twice.

Reply:
thanks for the feedback @Chavithra , done that!